gui.py

```diff
@@ -284,34 +257,20 @@ def import_settings(self):
         """
         file_path = filedialog.askopenfilename(
                 parent=self.main_frame, title="Select Settings File",

```

Comment:
```suggestion
            parent=self.main_frame, title="Select Settings File",
```
Reduce indent for params (in conjunction with the next change)